### PR TITLE
VAGOV-4408: Adding staff profiles to paragraphs.

### PR DESCRIPTION
**Pull request by ethanteague**
## Description
Allows user to add staff profile paragraph to detail page

## Testing done
Visual

## Screenshots
Screenshot_2019-07-30_20-49-00
Screenshot_2019-07-30_20-52-16
Screenshot_2019-07-30_20-52-34

## Acceptance criteria
- [ ] Adding a staff profile paragraph on a drupal detail page displays on frontend after build

### CODE REVIEW

File: src/platform/site-wide/sass/shame.scss [hunk 1]
```diff
@@ -527,3 +527,7 @@ img[data-srcset] {
   list-style: none !important;
 }
 
+.circular-profile-image {
+  border-radius: 250px;
```
Comment (ahay-agile6) [inline src/platform/site-wide/sass/shame.scss]: did `border-radius: 50%` not work?

File: src/site/includes/bioParagraph.drupal.liquid [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+        vads-u-margin-bottom--0p5">
+      {{ bio.fieldOffice.entity.entityLabel }}
+    </p>
+    {% endif %}
+    {% if bio.fieldEmailAddress %}
+    <p class="
+        vads-u-font-weight--normal
+        vads-u-margin--0
+        vads-u-margin-bottom--0p5">
+      <b>Email:</b> <a target="blank"
```
Comment (ahay-agile6) [inline src/site/includes/bioParagraph.drupal.liquid]: do we need the `_` for `target="_blank"`?

File: src/site/includes/bioParagraph.drupal.liquid [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+        href="{{ bio.fieldEmailAddress }}">{{ bio.fieldEmailAddress }}</a>
```
Comment (ahay-agile6) [inline src/site/includes/bioParagraph.drupal.liquid]: should this be a `mailto` href?

**Review by aurora-a-k-a-lightning — COMMENTED** [review]
some questions

File: src/site/includes/bioParagraph.drupal.liquid (hunk 2 above)
Comment (ethanteague) [reply src/site/includes/bioParagraph.drupal.liquid]: Yeah, Jeff Brauer wants gmail to open in a new window, @ahay-agile6

File: src/site/includes/bioParagraph.drupal.liquid (hunk 3 above)
Comment (ethanteague) [reply src/site/includes/bioParagraph.drupal.liquid]: YES! Thank you for catching that, @ahay-agile6

File: src/platform/site-wide/sass/shame.scss (hunk 1 above)
Comment (ethanteague) [reply src/platform/site-wide/sass/shame.scss]: Lol - I had no idea until you suggested, and yes, that does work, @ahay-agile6

**Review by aurora-a-k-a-lightning — APPROVED** [review]

**Pull request merged**